mne/viz/misc.py

```diff
@@ -418,6 +442,19 @@ def plot_bem(subject=None, subjects_dir=None, orientation='coronal',
     See Also
     --------
     mne.viz.plot_alignment
+
+    Notes
+    -----
+    Images are plotted in MRI voxel coordinates.
+
+    For a given slice index, all source points are shown that are halfway
+    between the previous slice and the given slice, and halfway between the
+    current slice and the next slice. For large slice decimations, this can
+    make some source points appear outside the BEM contour, which is shown
+    for the given slice index. For example, in the case where the single
+    midpoint slice is used ``slices=[128]``, all source points will be shown
+    on top of the midpoint MRI slice with the BEM boundary drawn for that
+    slice.
```

Comment:
```suggestion
    If ``brain_surfaces`` is not ``None``, the surface points that are plotted
    are those that fall within one-half of the inter-slice distance in either
    direction from the displayed slice. For large slice decimations, this can
    make some surface points appear outside the BEM contour (which is drawn
    where it intersects the displayed slice).
```

Reply:
@larsoner not sure if this captures everything you were trying to say...